### PR TITLE
Remove Thermal Regulators

### PR DESCRIPTION
**Pull request by KangTut**
## About The Pull Request
Removes the thermal regulator from spacesuits and hardsuits as well as any associated code

## Why It's Good For The Game
There isn't anything fun about having to keep track of a suits power charge. Going space exploring shouldn't mean that you either need to grab and charge a high tech cell or carry several backups. It was just more pointless and tedious crap added for no adequate reason.

## Changelog
-Removes the need to toggle your spacesuit's ability to protect you from space
-Removes power cells and anything related to them from space and hardsuits
-Removes EMP effects on hardsuits

### CODE REVIEW

**Review by TheChosenEvilOne — COMMENTED** [review]
just one thing, otherwise seems good.

File: code/datums/action.dm [hunk 1]
```diff
@@ -277,30 +277,11 @@
 		H.toggle_welding_screen(owner)
 
 /datum/action/item_action/toggle_spacesuit
```
Comment (TheChosenEvilOne) [inline code/datums/action.dm:279]: remove this datum completely.

**Review by TheChosenEvilOne — CHANGES_REQUESTED** [review]
one more thing.

File: code/datums/action.dm (hunk 1 above)
Comment (KangTut) [reply code/datums/action.dm]: Thanks for helping me with this.

File: code/modules/clothing/spacesuits/_spacesuits.dm [hunk 2]
```diff
@@ -39,238 +37,10 @@
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 0, BIO = 100, RAD = 50, FIRE = 80, ACID = 70)
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
 	cold_protection = CHEST | GROIN | LEGS | FEET | ARMS | HANDS
-	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT_OFF
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
```
Comment (EdgeLordExe) [inline code/modules/clothing/spacesuits/_spacesuits.dm:41]: this seems to be doubled for no reason

**Review by TheChosenEvilOne — APPROVED** [review]
lgtm

**Pull request merged**